README.md

```diff
@@ -3,21 +3,8 @@
 This repository is a demo projection for the HTML editor [roosterjs](https://github.com/microsoft/roosterjs).
 It shows how to build user interface on top of roosterjs based on React.
 
-The repository is not actively maintained. To build your UI with rooserjs, we recommend building with your own code
-rather than directly importing code from this repository.
+The repository is not actively maintained. For a newer version of roosterjs-react, please refer to [roosterjs-react](https://github.com/microsoft/roosterjs/tree/master/packages-ui)
+pacakge in [roosterjs](https://github.com/microsoft/roosterjs) repository. We now provide a new react component "Rooster" as a react wrapper of roosterjs.
```

Comment:
Couple of typo fixes:
"please refer to the [roosterjs-react](https://github.com/microsoft/roosterjs/tree/master/packages-ui)
package in the [roosterjs]"